app/pages/dev/dev_page.php

```diff
@@ -44,4 +44,43 @@ function DevButton($href, $label)
     <ul>
         <?= DevButton("/dev/google_calendar", "Google Calendar") ?>
     </ul>
-</nav>
\ No newline at end of file
+</nav>
+
+
+<b>PWA debug</b>
+<nav>
+    <ul>
+        <li><button id="pwa-support">display mode</button></li>
+        <li><button id="notification-support">check notification</button></li>
+    </ul>
+</nav>
```

Comment:
buttons in contrast outline ?

Reply:
Doesn't really matter, it's the dev page :)

Reply:
Let's hope we can remove this soon